src/main/resources/templates/rules.html

```diff
@@ -0,0 +1,8 @@
+<div class="content">
+    <div class="col-12">
+        <h1 class="text-center">Active Rules File</h1>
```

Comment:
I would like the rules.path to be displayed on this page, together with the rules. Also, since URL can also be used maybe file is not the best header? How about only "Active rules"?

Reply:
Done.